tools/accuracy_checker/tests/test_dlsdk_launcher.py

```diff
@@ -169,12 +180,15 @@ def test_dlsdk_onnx_import(self, data_dir, models_dir):
         dlsdk_test_model.predict([{'data': input_blob.astype(np.float32)}], [image.metadata])
         assert dlsdk_test_model.output_blob == 'fc3'
 
-
-
+@pytest.mark.skipif(ng is None and not has_layers(), reason='no functionality to set affinity')
```

Comment:
How could this happen? Shouldn't at least one of these always be present?

Reply:
if we use package, then yes both them present, if we build OV from source and follow original build instruction, then we do not have ngraph python, because it has own instruction how it should be installed (people who does not know about it or do not want work install it, can skip this).

Reply:
OK.